src/index.js

```diff
@@ -17,7 +20,25 @@ class ProjectionGridReact extends React.Component {
   }
 
   render() {
-    const model = this.core.compose({ config: _.omit(this.props, 'projections') });
+    const { classes, data, caption, cols, primaryKey, sorting, tfoot } = this.props;
+
+    const model = this.core.compose({
+      config: {
+        classes: utils.compact([...classes, this.props.className]),
+        data,
+        caption,
+        cols,
+        primaryKey,
+        sorting,
+        tfoot,
+      },
+      state: this.gridState,
+      dispatch: (reducer, ...args) => {
+        this.gridState = reducer(this.gridState, ...args);
```

Comment:
I just recalled this `state` & `dispatch` should be the default implementation. We need another 2 props to override them to integrate with user's Redux store.